en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md

```diff
@@ -120,7 +142,31 @@ The **apictl** can export an API as an archive from a lower environment (i.e., d
 
      For more information, see [Export an API]({{base_path}}/Learn/APIController/migrating-apis-to-different-environments/#export-an-api).
 
-## Preparing an API Project for CI/CD
+<a name="D"></a>
+### (D.) - Initialize the project using a Swagger/OpenAPI specification
+
+Execute the following command to directly generate the `PetstoreAPI` project using a Swagger/OpenAPI specification.
+
+!!! example
+    ```bash
+    apictl init PetstoreAPI --oas path/to/petstore.yaml
+    ```
+
+- This generates an API project in the `PetstoreAPI` directory using the provided specification. This project can be directly 
+imported into the API Manager.
+- The **apictl** allows further customization to the project initialization using a template file. Organization-specific common details can be added into this template file and shared across developers to increase productivity.
+- To further finetune API creation, an additional API Definition file can be used. This definition file supports detecting environment variables during the creation process. It can be combined with scripting to develop powerful tools for 
+automating API Project creation.
+- Using this method, the Swagger/OpenAPI specification becomes a single source of truth for API deployment. By combining 
+templating and the definition file, the automation servers can be configured to initialize API Projects from Swagger/OpenAPI 
+specifications and also have custom parameter files. This reduces human intervention and boosts productivity.
+- For example, when an organization depends on a microservices architecture, this method can be utilized to create an automated pipeline to move Swagger/OpenAPI specifications to upper environments.
+
+For more information on initializing an API Project using OpenAPI/Swagger Specification, see 
+[Initialize an API Project]({{base_path}}/Learn/APIController/importing-apis-via-dev-first-approach/#initialize-an-api-project).
+
+<a name="E"></a>
+### (E.) - Prepare an API project for CI/CD
 
 1.  Extract the content (API will be exported as an archive to the 
```

Comment:
This step 1 is needed only if you followed, step C before. Add **`Rename it to SwaggerPetstore for easy reference.`** to the same step1.

Keep step2 first sentence as it is. Remove the sentence **`Rename it to `SwaggerPetstore` for easy reference.`**

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/677/commits/eb59e1a0122cdb90be9152a172194ca6a78ca83c